mysite/wsgi.py

```diff
@@ -0,0 +1,11 @@
+"""WSGI config for mysite project."""
+
+
+# pylint: disable=invalid-name
```

Comment:
This is the wrong place. `os` name is not invalid. Put it where the linter complaint exists.